GTMAppAuthSwift/Sources/KeychainStore.swift

```diff
@@ -79,6 +79,8 @@ public final class KeychainStore: NSObject {
     self.itemName = itemName
     self.keychainAttributes = keychainAttributes
     self.keychainHelper = keychainHelper
+
+    NSKeyedUnarchiver.setClass(AuthState.self, forClassName: "GTMAppAuthFetcherAuthorization")
```

Comment:
Should we define a constant for "GTMAppAuthFetcherAuthorization"?

Reply:
Good idea.